src/Network/Network/help/New-AzApplicationGatewayFirewallGroupByVariable.md

```diff
@@ -0,0 +1,77 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.Network.dll-Help.xml
+Module Name: Az.Network
+online version: https://learn.microsoft.com/powershell/module/az.network/new-azapplicationgatewaygroupbyvariable
+schema: 2.0.0
+---
+
+# New-AzApplicationGatewayFirewallGroupByVariable
+
+## SYNOPSIS
+Creates a new GroupByVariable for the application gateway Firewall GroupByUserSession.
```

Comment:
Match this to previous definition and cases. Above we say application gateway firewall custom rule